ursgal/unimod_mapper.py

```diff
@@ -65,8 +66,13 @@ def _parseXML(self, xmlFile = None):
                 for event, element in unimodXML:
                     if event == b'start':
                         if element.tag.endswith('}mod'):
+                            try:
+                                unimod_id = element.attrib['record_id']
+                            except KeyError:
+                                unimod_id = extra_unimod_id
```

Comment:
unimod_id = 'u{0}'.format(extra_unimod_id)

the u prevents any overlap with unimod IDs and also allows us to easily distinguish unimod IDs from user-defined mods.